src/components/BottomNav/index.tsx

```diff
@@ -60,7 +66,26 @@ export const BottomNav: React.FC<BottomNavProps> = ({
           value="/user/store"
           icon={<StorefrontIcon />}
         ></BottomNavigationAction>
+        <BottomNavigationAction
+          label="More"
+          showLabel={true}
+          value="more"
+          icon={<MoreHorizIcon />}
+          onClick={handleClick}
+        ></BottomNavigationAction>
       </BottomNavigation>
+      <Menu
+        anchorEl={anchorEl}
+        keepMounted
+        open={Boolean(anchorEl)}
+        onClose={handleClose}
+        transitionDuration={200}
+      >
+        <MenuItem>Menu 1</MenuItem>
```

Comment:
I suppose we should figure out what actually needs to be here. I don't really like having a "more" button because it doesn't have much info on what the button does (like the hamburger).  

To be discussed